bootable-jar/runtime/src/main/java/org/wildfly/core/jar/runtime/CmdUsage.java

```diff
@@ -48,6 +48,9 @@ public static void init() {
         addArguments(CommandLineConstants.SECMGR);
         instructions.add(BootableJarLogger.ROOT_LOGGER.argSecurityManager());
 
+        addArguments(CommandLineConstants.STABILITY + "=<value>");
+        instructions.add(BootableJarLogger.ROOT_LOGGER.argStability());
```

Comment:
Would it be possible for the Bootable JAR to get the available stability levels from ProductConfig.getStabilitySet()?

In addition, we have a kind of restriction here where the stability levels should be hidden when there is only one permissible stability level. See https://github.com/wildfly/wildfly-core/blob/b20d4c06b41542316af51ff213bb564f2574d3d3/server/src/main/java/org/jboss/as/server/CommandLineArgumentUsageImpl.java#L83-L86 as a reference. CC @pferraro

Reply:
@yersan , thank-you. I adjusted to comply with what we do in the server.